File: 240602/출석 부르기 2/calling-attendance-2.py
while True:
    n = int(input())
    
    if n not in [1, 2, 3, 4]:
        print("Vacancy")
        break
    elif n == 1:
        print("John")
    elif n == 2:
        print("Tom")
    elif n == 3:
        print("Paul")
    elif n == 4:
        print("Sam")

File: 251026/출석 부르기 2/calling-attendance-2.cpp
#include <iostream>
#include <map>
#include <string>
using namespace std;

int main() 
{
    // Please write your code here.
    int n;
    map<int, string> students = {
        {1, "John"}, 
        {2, "Tom"}, 
        {3, "Paul"}, 
        {4, "Sam"}
    };
    cin >> n;

    while(n >= 1 && n <= 4)
    {
        cout << students[n] << endl;
        cin >> n;
    }
    cout << "Vacancy";
    return 0;
}